src/BqPropertiesManager.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Meisterwerk\BqUtils;
+
+class BqPropertiesManager
+{
+    private const BQ_API_PATH_V1 = 'https://rentshop.booqable.com/api/1/';
+    private const BQ_API_PATH_V3 = 'https://rentshop.booqable.com/api/3/';
+
+    private BqRestManager $bqRestManagerV1;
+
+    private BqRestManager $bqRestManagerV3;
+
+    public function __construct(string $apiKey)
+    {
+        $this->bqRestManagerV1 = new BqRestManager($apiKey, self::BQ_API_PATH_V1);
+        $this->bqRestManagerV3 = new BqRestManager($apiKey, self::BQ_API_PATH_V3);
+    }
+
+    /**
+     * @param array $propertyFields example: $propertyFields = ['identifier' => 'sprache', 'name' => 'Sprache',]
+     * @throws BqRequestException
+     */
+    public function createOrUpdateProperty(string $bqOrderId, string $value, array $propertyFields): void
```

Comment:
Bitte `propertyFields` zu `propertyQuery` umbenennen und dafür eine Klasse machen (`BqOrderPropertyQuery`).  
Diese Klasse dann grad zusammen mit dem Manager in ein neues Package `OrderProperties`  verschieben.

Reply:
Parameteter `$bqOrderId` bitte zu `orderId` umbenennen. Mach es nicht unnötig kompliziert.